src/dbstore/originalTxsData.ts

```diff
@@ -95,7 +96,7 @@ export async function queryOriginalTxsData(
     let sql = `SELECT * FROM originalTxsData`
     const sqlSuffix = ` ORDER BY cycle ASC, timestamp ASC LIMIT ${limit} OFFSET ${skip}`
     const values: number[] = []
-    if (startCycle && endCycle) {
+    if (startCycle >= 0 && endCycle >= 0) {
```

Comment:
I think we should instead change it to `isUndefined / isNaN` check instead. because we may want query data without cycle parameters.